### PR TITLE
Fuel v3

### PR DESCRIPTION
**Pull request by frodrigo**

### CODE REVIEW

File: analysers/analyser_merge_fuel_IT.py [hunk 1]
```diff
@@ -62,9 +62,8 @@ def __init__(self, config, logger = None):
...
-    def normalizeString(self, str):
-        search = [ 'A', 'E', 'ED', 'DI', 'DIS-CAR', 'SOCIETA\'', 'RESPONSABILITA\'', 'SNC', 'SAS' ]
-        replace = [ 'a', 'e', 'ed', 'di', 'Dis-car', 'Società', 'Responsabilità', 'S.N.C.', 'S.A.S.' ]
-        return ' '.join(map(lambda x: replace[search.index(x)] if x in search
-            else x.title(), str.replace('"', ' ').replace(',', ' ').split()))
-
+    WORDS_MAP = {'A': 'a', 'E': 'e', 'ED': 'ed', 'DI': 'di', 'DIS-CAR':'Dis-car', 'SOCIETA\'': 'Società',
+        'RESPONSABILITA\'': 'Responsabilità', 'SNC': 'S.N.C.', 'SAS': 'S.A.S.'}
+    def normalizeString(self, s):
+        s = s.replace('"', ' ').replace(',', ' ').replace('  ', ' ').title()
```
Comment (fansanelli) [inline analysers/analyser_merge_fuel_IT.py]: extra spaces are trimmed away by the split. title should go in next row.
I will send a patch based on this take into account your changes

File: analysers/analyser_merge_fuel_IT.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def __init__(self, config, logger = None):
                     static2 = {'source': self.source},
                     mapping1 = {
                         'ref:mise': 'idImpianto',
-                        'operator': lambda res: normalizeString(res[u'Gestore']),
+                        'operator': lambda res: self.normalizeString(res['Gestore']),
```
Comment (fansanelli) [inline analysers/analyser_merge_fuel_IT.py]: the u... should stay or should go?

Comment (frodrigo) [reply analysers/analyser_merge_fuel_IT.py]: no `u''` in python 3. All the `u''` in Osmose code from python 2.

File: analysers/analyser_merge_fuel_IT.py (hunk 1 above)
Comment (frodrigo) [reply analysers/analyser_merge_fuel_IT.py]: Ok. Please, fix it.

Comment (fansanelli) [reply analysers/analyser_merge_fuel_IT.py]: done... was easier to send it than to explain it. :)

File: analysers/analyser_merge_fuel_IT.py (hunk 2 above)
Comment (fansanelli) [reply analysers/analyser_merge_fuel_IT.py]: I'm not that python expert... since 3 u is not needed, right?

Comment (frodrigo) [reply analysers/analyser_merge_fuel_IT.py]: Yes. It is the new default.

Comment (fansanelli) [reply analysers/analyser_merge_fuel_IT.py]: ok. I'll commit a fix now

Comment (frodrigo) [reply analysers/analyser_merge_fuel_IT.py]: You can just merge my PR.

Comment (fansanelli) [reply analysers/analyser_merge_fuel_IT.py]: sorry for not doing it in the right way. my bad

**Pull request merged**